docs/_sass/_base.scss

```diff
@@ -173,3 +173,24 @@ figcaption {
         }
     }
  }
+
+ .wrapper__code-examples {
+     & .wrapper__code-example {
+        margin-bottom: 64px;
+        &:last-of-type {
+            margin-bottom: 24px;
+        }
+     }
+    /* Overwrite default code-block styling */
+    & .code-example {
+        & pre, & .highlight {
+            background: transparent;
+        }
+    }
+    & .code-example.pass {
```

Comment:
nit: the code style in this file suggests that `& .` should be `&.` and that there should be a new line before each of the blocks. same for `&:` (applies to all code changes in this file)